src/Hosting/Hosting/test/HostTests.cs

```diff
@@ -81,6 +82,56 @@ public void CreateDefaultBuilder_EnablesValidateOnBuild()
             Assert.Throws<AggregateException>(() => hostBuilder.Build());
         }
 
+        [Fact]
+        public void CreateDefaultBuilder_ConfigJsonDoesNotReload()
+        {
+            Environment.SetEnvironmentVariable("DOTNET_HOSTBUILDER_CONFIG_RELOAD", "false");
+            Environment.CurrentDirectory = Path.GetTempPath();
```

Comment:
Changing CurrentDirectory is also not a good idea. Why did you need to do that? So you'd have somewhere to write your appsettings.json? It would be better to change the host's ContentRoot to the temp path.
https://docs.microsoft.com/en-us/aspnet/core/fundamentals/host/generic-host?view=aspnetcore-3.1#contentrootpath

Reply:
Yes just trying to sandbox. I didn't know content root would redirect configuration flatfile loading, I was just worried about it since the filepath is [hard coded without prefix](https://github.com/dotnet/extensions/blob/master/src/Hosting/Hosting/src/Host.cs#L74-L75). Thanks, I'll update this.